snippets/scripts/compass-github-importer/index.js

```diff
@@ -123,8 +123,80 @@ async function listAllOrgs() {
     return instanceOrganizations;
 }
 
+async function createOrgWebhooks(orgs) {
+    for (const org of orgs) {
+        const webhookName = `${org.login} webhook`
+        const response = await makeGqlRequest({
+            query: `
+        mutation MyMutation {
+  compass @optIn(to: "compass-beta"){
+    createIncomingWebhook(input: {cloudId: "${CLOUD_ID}", name: "${webhookName}", source: "github_enterprise_server"}) {
+      success
+      errors {
+        message
+        extensions {
+          statusCode
+          errorType
+        }
+      }
+      webhookDetails {
+        id
+      }
+    }
+  }
+}`,
+        });
+
+        const maybeId = response?.data.compass?.createIncomingWebhook?.webhookDetails?.id
+        const isSuccess = response?.data.compass?.createIncomingWebhook?.success
+
+        if (!isSuccess || !maybeId) {
+            console.error(`error creating incoming webhook: `, JSON.stringify(response));
+            throw new Error('Could not create incoming webhook');
+        }
+
+        console.log(chalk.green(`New compass incoming webhook for organization "${org.login}" created`))
+
+        const webhookId = maybeId.substring(maybeId.lastIndexOf('/') + 1)
+        const webhookUrl = `https://${TENANT_SUBDOMAIN}.atlassian.net/gateway/api/compass/v1/webhooks/${webhookId}`
+
+
+        const githubResponse = await axios.post(
+            `${GITHUB_URL}/api/v3/orgs/${org.login}/hooks`,
+            {
+                name:'web',
+                config: {
+                    url: `${webhookUrl}`,
+                    content_type: 'json',
+                },
+                events: ['deployment_status', 'workflow_run', 'push']
+            },
+            {
+                headers: {
+                    'Authorization': `Bearer ${ACCESS_TOKEN}`,
+                    'Accept': 'application/vnd.github+json',
+                },
+            }
+        );
+
+        if(githubResponse.status !== 201){
+            console.error(`error creating github webhook: `, JSON.stringify(response));
+            throw new Error('Could not create github webhook');
+        }
+
+        console.log(chalk.green(`New GitHub webhook for organization "${org.login}" created`))
+    }
+}
+
 async function listAllRepos() {
     const instanceOrganizations = await listAllOrgs()
+
+    try {
+        await createOrgWebhooks(instanceOrganizations)
```

Comment:
Would this create a new webhook on every call to `listAllRepos`?
* Can we confirm that this step would be skipped if this is set for a Dry Run?
* Should we separate `createOrgWebhooks` from `listAllRepos`? It seems that they serve two different needs and users may want to execute them separately.

We also need some updates to the Readme.md, to clarify that there's now an operation available to create an inbound webhook in Compass and connecting that to the GitHub organization.